src/pages/home/sidebar/ChatSwitcherView.js

```diff
@@ -428,7 +428,7 @@ class ChatSwitcherView extends React.Component {
                 alternateText: personalDetail.login,
                 searchText: personalDetail.displayName === personalDetail.login ? personalDetail.login
                     : `${personalDetail.displayName} ${personalDetail.login}`,
-                icon: personalDetail.avatarURL,
+                icon: Array.isArray(personalDetail.avatarURL) ? personalDetail.avatarURL : [personalDetail.avatarURL],
```

Comment:
Why would `personalDetail.avatarURL` be an array? Shouldn't a personal detail only have one avatar?

Reply:
should be using `_.isArray()` not `Array.isArray()` for consistency, but I think it makes no sense.